docs/destinations/forwarding-events/snowbridge/getting-started/distribution-and-deployment.md

```diff
@@ -15,7 +15,7 @@ import CodeBlock from '@theme/CodeBlock';
 
 There are two distributions of Snowbridge.
 
-<Tabs groupId="distribution">
+<Tabs groupId="cloud">
```

Comment:
This is not technically correct. The first distribution is cloud-agnostic, and the second one is AWS-specific. Or is this to pre-select the right tab when navigating from other pages where AWS was selected?

Reply:
> Or is this to pre-select the right tab when navigating from other pages where AWS was selected?
This, yeah. I wasn't sure about this one either. I'm happy to undo this one if the semantics are better than the usability gain?

Reply:
Having thought about it a bit more, I suppose I can be convinced with the usability argument. Let’s keep your change :)